services/galley/test/integration/API/Util.hs

```diff
@@ -123,6 +123,11 @@ createTeamConv g u tinfo us name = do
               ) <!! const 201 === statusCode
     fromBS (getHeader' "Location" r)
 
+createO2OTeamConv :: Galley -> UserId -> UserId -> Maybe Text -> TeamId -> Http ResponseLBS
+createO2OTeamConv g u1 u2 n tid = do
```

Comment:
`createOne2OneTeamConv`? `create1to1TeamConv` ? Abbreviating `One` as `O` is odd.

Reply:
Right, I'll change that - thanks!